--- A1	2024-04-29 09:50:52.398841254 +0000
+++ A2	2024-04-29 09:50:52.398841254 +0000
@@ -1,5 +1,5 @@
 Linux is an open-source operating system that is widely used across various platforms including servers, desktops, and embedded systems.
 It is known for its stability, security, and flexibility, making it a popular choice for developers and system administrators.
-Linux operates under the GNU General Public License (GPL), allowing user to modify and distribute the software freely.
+Linux operates under the GNU General Public License (GPL), allowing users to modify and distribute the software freely.
 Its kernel, created by Linus Torvalds in 1991, is central to its functioning, facilitating hardware and software communication.
 Linux supports a vast ecosystem of distributions, each tailored for specific needs, such as Ubuntu, Fedora, and Debian, enhancing its adaptability and usability.
\ No newline at end of file
